### ISSUE TITLE
0.0.24 postgres+south migrations into clean database don't work

### ISSUE THREAD

**Issue by black3r** [django-wiki/django-wiki#394]
**Software versions**:
```
Python 2.7.8
```
```
Django==1.6.10
Markdown==2.5.2
Pillow==2.7.0
South==1.0
django-classy-tags==0.6.1
django-mptt==0.6.0
django-nyt==0.9.6.post1
django-sekizai==0.8.1
psycopg2==2.6
six==1.9.0
sorl-thumbnail==12.2
wiki==0.0.24.2
wsgiref==0.1.2
```

**Steps to reproduce**:

 - Create a new project using django-admin.py 
 - Set up wiki in settings.py
 - Set up postgres_psycopg2 database in settings.py
 - python manage.py syncdb
 - python manage.py migrate

**Expected Behavior**: 
everything works correctly, just like it does with sqlite and mysql

**What actually happens**: 
migrate throws this error:
```
 > wiki:0007_auto__add_articlesubscription
FATAL ERROR - The following SQL query failed: ALTER TABLE "wiki_articlesubscription" ADD CONSTRAINT "subscription_ptr_id_refs_id_4d6e61f2" FOREIGN KEY ("subscription_ptr_id") REFERENCES "notify_subscription" ("id") DEFERRABLE INITIALLY DEFERRED;
The error was: relation "notify_subscription" does not exist

Error in migration: wiki:0007_auto__add_articlesubscription
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/django/core/management/__init__.py", line 399, in execute_from_command_line
    utility.execute()
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/django/core/management/base.py", line 242, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/django/core/management/base.py", line 285, in execute
    output = self.handle(*args, **options)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/south/management/commands/migrate.py", line 111, in handle
    ignore_ghosts = ignore_ghosts,
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/south/migration/__init__.py", line 220, in migrate_app
    success = migrator.migrate_many(target, workplan, database)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/south/migration/migrators.py", line 256, in migrate_many
    result = migrator.__class__.migrate_many(migrator, target, migrations, database)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/south/migration/migrators.py", line 331, in migrate_many
    result = self.migrate(migration, database)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/south/migration/migrators.py", line 133, in migrate
    result = self.run(migration, database)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/south/migration/migrators.py", line 114, in run
    return self.run_migration(migration, database)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/south/migration/migrators.py", line 85, in run_migration
    south.db.db.execute_deferred_sql()
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/south/db/generic.py", line 318, in execute_deferred_sql
    self.execute(sql)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/south/db/generic.py", line 282, in execute
    cursor.execute(sql, params)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/django/db/backends/util.py", line 69, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/django/db/backends/util.py", line 53, in execute
    return self.cursor.execute(sql, params)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/django/db/utils.py", line 99, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/roman/.virtualenvs/testwiki/lib/python2.7/site-packages/django/db/backends/util.py", line 53, in execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: relation "notify_subscription" does not exist
```

**Workaround**: 

- wait for the error to happen
- python manage.py migrate --fake wiki 0007 
- python manage.py migrate 

does not happen in master because migrations are reset in master

**Comment by benjaoming** [comment]
Thanks @black3r for giving details necessary to reproduce, I was wondering about how to reproduce in #224. So, I guess this is a candidate for a new .3 release...

**Comment by black3r** [comment]
this seems to be fixed in 0.0.24.4 (also it shouldn't affect upgrading to 0.1 even if it wasn't)

**Comment by benjaoming** [comment]
Oh yes, true! I didn't experience anymore, neither. Thanks for getting back! I'll go ahead and close this now then.
